controllers/articlesCommentController.js

```diff
@@ -57,7 +70,7 @@ async function findCommentController(req, res) {
       return failureResponse(res, COMMENT_NOT_FOUND, NOT_FOUND_CODE);
     }
     const comment = comments.map((thread) => {
-      thread.threads = thread.threads.length;
+      thread.dataValues.threads = thread.threads.length;
```

Comment:
`thread.dataValues.threads` why the change here?

Reply:
I added dataValues because without dataValues it was not giving the desired result. It was lucky that implmented it, but when you said we would get the sane value without dataValues Kabril removed it. When I merged with mine it was not having the same effect that was there before

Reply:
Why is this so? It worked well for @Luckzman and @ibraheemkabir. Why is it different for you?

Reply:
dataValues was there when lucky used it, Ibrahim didn't use it he only removed all dataValues in the file. He had no use of it.

Reply:
Without dataValues
<img width="616" alt="screen shot 2019-02-07 at 7 42 50 pm" src="https://user-images.githubusercontent.com/32283153/52434859-a6d5a080-2b10-11e9-9fb7-bb3dcc66dbfe.png">

Reply:
With dataValues (Desired Result)
<img width="683" alt="screen shot 2019-02-07 at 7 42 31 pm" src="https://user-images.githubusercontent.com/32283153/52434884-b81ead00-2b10-11e9-81a0-a323ca1a593e.png">

Reply:
The thread count is where the effect is seen

Reply:
Cool